tools/storyteller/Storyteller.java

```diff
@@ -197,6 +197,7 @@ YearMonth getLastMonth(Instant instant) {
   }
 
   public void startup(String project) {
+    writer.addStories(getUnsharedStories());
```

Comment:
Maybe you should put this in the constructor, so it's called only once. Would that fix the issue? Then you don't need to `allStories = new ArrayList<>();` in `addStories()`.